lib/utils/functionInfoUtils.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+module.exports = {
+  aws: {
```

Comment:
🤔 yes, we discussed that yesterday in the call and came up with that quick solution (also discussed the possibility to move it into another file).